resources/sass/themes/modules/_themeable.scss

```diff
@@ -231,7 +231,7 @@ a.next:link .title,
 a.next:visited .title,
 a.previous:link .title,
 a.previous:visited .title {
-    color: var(--base-color);
+    color: light-dark(var(--base-color), var(--white));
```

Comment:
I wonder if just deleting this rule entirely would cause this to inherent from the more general rules for links?

If not, I think these links should follow the color scheme of other links, so instead of `var(--white)` the dark option should be `var(--yellow-green)`